iothub/device/src/Pipeline/RetryDelegatingHandler.cs

```diff
@@ -82,16 +98,19 @@ public override async Task SendTelemetryBatchAsync(IEnumerable<TelemetryMessage>
             if (Logging.IsEnabled)
                 Logging.Enter(this, messages, cancellationToken, nameof(SendTelemetryAsync));
 
+            await VerifyIsOpenAsync(cancellationToken).ConfigureAwait(false);
+            using var operationCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, _cancelPendingOperationsCts.Token);
```

Comment:
This is the linked cancellation token that will be signaled for closure if `CloseAsync()` is called

Reply:
Updating the formatting marked this comment as outdated. Resolving this and adding a new one.